sql/setup.sql

```diff
@@ -1,2 +1,72 @@
 -- Use this file to define your SQL tables
 -- The SQL in this file will be executed when you run `npm run setup-db`
+DROP TABLE IF EXISTS combo;
+DROP TABLE IF EXISTS authors;
+DROP TABLE IF EXISTS books;
+
+CREATE TABLE authors (
+  id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR NOT NULL,
+  dob DATE,
+  pob VARCHAR
+);
+
+CREATE TABLE books (
+  id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  title VARCHAR NOT NULL,
+  released INT NOT NULL
+);
+
+CREATE TABLE combo (
```

Comment:
call this table ```authors_books``` — A) that's the convention for join tables B) that way you can have more than one join table (what if you have a publishing company table and need to associate authors with publishing companies?) C) it makes clear WHICH two tables are being joined, especially useful since each column is just a number

(or books_authors, doesn't matter)